getMajorMinorIndex.js

```diff
@@ -0,0 +1,24 @@
+var Major_reference = require("./MajorMinor.js").MajorColors;
+var Minor_reference = require("./MajorMinor.js").MinorColors;
+function getMajorIndex(majorColor){
+    
```

Comment:
getMajorIndex and getMinorIndex are similar. Can we extract the common part and reduce the duplication?